test/expected/plan_ordered_append-11.out

```diff
@@ -806,6 +876,120 @@ ORDER BY time DESC;
+         ->  Index Scan using _hyper_7_24_chunk_space_time_idx on _hyper_7_24_chunk (actual rows=6720 loops=1)
+         ->  Index Scan using _hyper_7_26_chunk_space_time_idx on _hyper_7_26_chunk (actual rows=13440 loops=1)
+         ->  Index Scan using _hyper_7_28_chunk_space_time_idx on _hyper_7_28_chunk (actual rows=6720 loops=1)
+         ->  Index Scan using _hyper_7_30_chunk_space_time_idx on _hyper_7_30_chunk (actual rows=13440 loops=1)
+(14 rows)
+
+-- test hypertable with space partitioning ORDER BY multiple columns
+-- does not use ordered append
+:PREFIX SELECT
+  time, device_id, value
+FROM space
+ORDER BY time, device_id LIMIT 1;
+                                 QUERY PLAN                                  
+-----------------------------------------------------------------------------
+ Limit (actual rows=1 loops=1)
+   ->  Sort (actual rows=1 loops=1)
+         Sort Key: _hyper_7_24_chunk."time", _hyper_7_24_chunk.device_id
+         Sort Method: top-N heapsort  Memory: 25kB
+         ->  Append (actual rows=77766 loops=1)
+               ->  Seq Scan on _hyper_7_24_chunk (actual rows=6720 loops=1)
+               ->  Seq Scan on _hyper_7_26_chunk (actual rows=13440 loops=1)
+               ->  Seq Scan on _hyper_7_28_chunk (actual rows=6720 loops=1)
+               ->  Seq Scan on _hyper_7_30_chunk (actual rows=13440 loops=1)
+               ->  Seq Scan on _hyper_7_23_chunk (actual rows=6241 loops=1)
+               ->  Seq Scan on _hyper_7_25_chunk (actual rows=12482 loops=1)
+               ->  Seq Scan on _hyper_7_27_chunk (actual rows=6241 loops=1)
+               ->  Seq Scan on _hyper_7_29_chunk (actual rows=12482 loops=1)
+(13 rows)
+
+-- test hypertable with space partitioning ORDER BY non-time column
+-- does not use ordered append
+:PREFIX SELECT
+  time, device_id, value
+FROM space
+ORDER BY device_id, time LIMIT 1;
+                                 QUERY PLAN                                  
+-----------------------------------------------------------------------------
+ Limit (actual rows=1 loops=1)
+   ->  Sort (actual rows=1 loops=1)
+         Sort Key: _hyper_7_23_chunk.device_id, _hyper_7_23_chunk."time"
+         Sort Method: top-N heapsort  Memory: 25kB
+         ->  Append (actual rows=77766 loops=1)
+               ->  Seq Scan on _hyper_7_23_chunk (actual rows=6241 loops=1)
+               ->  Seq Scan on _hyper_7_24_chunk (actual rows=6720 loops=1)
+               ->  Seq Scan on _hyper_7_25_chunk (actual rows=12482 loops=1)
+               ->  Seq Scan on _hyper_7_26_chunk (actual rows=13440 loops=1)
+               ->  Seq Scan on _hyper_7_27_chunk (actual rows=6241 loops=1)
+               ->  Seq Scan on _hyper_7_28_chunk (actual rows=6720 loops=1)
+               ->  Seq Scan on _hyper_7_29_chunk (actual rows=12482 loops=1)
+               ->  Seq Scan on _hyper_7_30_chunk (actual rows=13440 loops=1)
+(13 rows)
+
+-- test LATERAL with correlated query
```

Comment:
should we also do a lateral test with space partitioned hypertable?

Reply:
this test is mostly in preparation for runtime exclusion